lib/main.dart

```diff
@@ -17,55 +16,143 @@ class Home extends StatelessWidget {
         centerTitle: true,
         title: Image.asset("assets/ikhlas.png"),
       ),
-      body: Container(
-        constraints: BoxConstraints.expand(width: 420, height: 162),
-        decoration: BoxDecoration(color: Colors.teal[300]),
-        padding: new EdgeInsets.all(16.0),
-        child: Column(
-          crossAxisAlignment: CrossAxisAlignment.start,
-          children: [
-            Row(
-              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      body: Column(
+        children: [
+          Container(
+            constraints: BoxConstraints.expand(width: 420, height: 162),
+            decoration: BoxDecoration(color: Colors.teal[300]),
+            padding: new EdgeInsets.all(16.0),
+            child: Column(
               crossAxisAlignment: CrossAxisAlignment.start,
-              children: <Widget>[
-                Column(
+              children: [
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
                   crossAxisAlignment: CrossAxisAlignment.start,
                   children: <Widget>[
-                    Text(
-                      "Next , Zohor",
-                      style: TextStyle(color: Colors.white, fontSize: 14),
-                    ),
-                    Text(
-                      '1:04 PM',
-                      style: TextStyle(color: Colors.white, fontSize: 32),
-                    ),
-                    Text(
-                      'Starts in 02:29:46',
-                      style: TextStyle(color: Colors.white, fontSize: 14),
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: <Widget>[
+                        Text(
+                          "Next , Zohor",
+                          style: TextStyle(color: Colors.white, fontSize: 14),
+                        ),
+                        Text(
+                          '1:04 PM',
+                          style: TextStyle(color: Colors.white, fontSize: 32),
```

Comment:
Please don't repeat the codes. Move this to generic method and call it here. If you have any doubt, you can ping me